### DISCUSSION TITLE
Pico-8 Games not loading, Please Help!

### DISCUSSION THREAD

**Discussion by Beees777** [eduardofilo/RG350_adam_image#315] [Q&A]
I've tried both .png format and the .p8 format, but i can't seem to get either format to load. Has anyone else had any trouble loading pico-8? I could really use some help/ advice.

**Comment by Medard22** [comment]
Hey, on my RG350M I had a good experience with the tac08 emulator. You can look for tac08.opk here https://0xcafed00d.itch.io/tac08-rg350

Now to make it work in SimpleMenu copy tac08.opk to /media/data/apps/   
After that open /media/data/local/home/.simplemenu/section_groups/home computers.ini and add tac08.opk to the execs line. it should look like this
execs = /media/data/apps/tac08.opk,/media/data/apps/fake08_gcw0.opk

Add .p8.png files to /media/sdcard/roms/PICO8 and start the from SimpleMenu it should work fine :)

**Reply by Medard22** [reply]
I just checked and I have no problems with fake08 either. The correct format should be <gamename>.p8.png in  /media/sdcard/roms/PICO8/ directory.

**Comment by eduardofilo** [comment]
@Beees777, have you tried with the two standalone emulators for Pico8 installed beside the RetroArch core that is used by default to run the games in this platform? You can change it by pressing Select in the ROM listings, like you can see in this entry of the wiki:
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-6.-Problem-solving#emulator-selection

@Medard22 Thanks for the suggestion for @Beees777, but that tac08 is already installed in Adam, together with retro8 RA core and face08 standalone emulator.

**Reply by eduardofilo** [reply]
There was a problem in RetroArch's configuration that explained why games in PNG format did not load (the integrated RA image viewer was launched). @Shatur reported it in issue #322 and fixed it in PR #323. It will be included in the next release, but in the meantime you can make the change to the RA configuration discussed in #322.

**Reply by JazzJohannes** [reply]
Also simplemenu was updated since v9.4 (now 10.X), and included auto-previews of .png roms (it's image at the same time as rom), so a new update with a new simplemenu would be ace!
